escodegen.js

```diff
@@ -2500,16 +2504,6 @@
         preserveBlankLines = options.format.preserveBlankLines && sourceCode !== null;
         extra = options;
 
-        if (sourceMap) {
```

Comment:
The `sourceMap` here shadows the outer `sourceMap`, so this has changed behaviour. Use this instead:

```js
if (sourceMap) {
  SourceNode = sourceMap.SourceNode;
}
```